fansi/shared/src/main/scala/fansi/Fansi.scala

```diff
@@ -644,16 +651,25 @@ object Color extends Category(offset = 3, width = 9){
     for(x <- 0 to 256)
     yield makeAttr(s"\u001b[38;5;${x}m", 17 + x)(s"Color.Full($x)")
 
-  val all = Vector(
+
+  val all: Vector[Attr] = Vector(
     Reset, Black, Red, Green, Yellow, Blue, Magenta, Cyan, LightGray, DarkGray,
     LightRed, LightGreen, LightYellow, LightBlue, LightMagenta, LightCyan, White
   ) ++ Full
+
 }
 
 /**
   * [[Attr]]s to set or reset the color of your background
+  * Color a encoded on 25 bit as follow :
+  * 0 : reset value
+  * 1 - 16 : 3 bit colors
+  * 17 - 273 : 8 bit colors
+  * 274 - 16 777 389 : 24 bit colors
```

Comment:
I have just copied what you told me on Friday ;)
But to be sure I wrote a test and you were right the first time.
I have noticed a mistake thanks to the tests.